sources/lib/Resources/config/services/pomm.yml

```diff
@@ -8,10 +8,16 @@ services:
         arguments: ['@pomm']
         tags:
             - { name: request.param_converter, converter: pomm.entity }
+
     pomm.serializer.normalizer:
         class: 'PommProject\SymfonyBridge\Serializer\Normalizer\FlexibleEntityNormalizer'
         tags:
-            - { name: serializer.normalizer }
+            - { name: serializer.normalizer, priority: 999 }
```

Comment:
If I recall well, it requires a priority to override the default normalizers provided by `symfony/serializer`.